spec/controllers/conferences_controller_spec.rb

```diff
@@ -26,6 +26,13 @@
     end
   end
 
+  describe 'GET #current' do
+    it 'redirects to first conference splashpage' do
+      current = Conference.first
+      expect(response).to redirect_to(conference_path(current.short_title))
```

Comment:
So I should add a line `get :current` just after the it 'redirects to first conference splashpage' do`